cadctap/cadctap/tests/data/help_query.txt

```diff
@@ -32,11 +33,12 @@ optional arguments:
   -i, --input-file INPUT_FILE
                         read query string from file (default is from STDIN),
                         location of file
+  -m, --maxrec MAXREC   Limit the number of returned records to this maximum
   -n                    use .netrc in $HOME for authentication
   --netrc-file NETRC_FILE
                         netrc file to use for authentication
   -o, --output-file OUTPUT_FILE
-                        write query results to file (default is to STDOUT)
+                        Write query results to file (default is to STDOUT)
```

Comment:
Because you changed in the case on this item, I'm going to mention that the case is inconsistent wtih the `-n`, '-q`, and `-s` help text.

This is a _very_ nitpicky comment, so deal with it as you wish.

Reply:
A total mishmash of cases. Fixed most of them.